pkg/operator/ceph/config/config.go

```diff
@@ -137,20 +137,6 @@ func SetOrRemoveDefaultConfigs(
 		}
 	}
 
-	// Apply Multus if needed
-	if clusterSpec.Network.IsMultus() {
-		logger.Info("configuring ceph network(s) with multus")
-		cephNetworks, err := generateNetworkSettings(clusterInfo.Context, context, clusterInfo.Namespace, clusterSpec.Network.Selectors)
-		if err != nil {
-			return errors.Wrap(err, "failed to generate network settings")
-		}
-
-		// Apply ceph network settings to the mon config store
-		if err := monStore.SetAll("global", cephNetworks); err != nil {
-			return errors.Wrap(err, "failed to network config overrides")
-		}
-	}
-
```

Comment:
Removed here and moved replacement to mon code because there were import chains when code existed here.